server/db.sql

```diff
@@ -0,0 +1,302 @@
+
+ALTER TABLE ONLY public.yarn ALTER COLUMN id SET DEFAULT nextval('public.yarn_id_seq'::regclass);
+
+--
+-- Insert Values into yarn table
+INSERT INTO public.yarn(project_id, yarn_brand, yarn_type, yarn_weight, recommended_hook_size, yarn_color, hook_size) 
+ VALUES 
+    (2, 'Loops and Threads Soft Classic', 'Acrylic', 'Medium (4)', '5.5mm', 'Sage', '4mm'),
+    (2, 'Loops and Threads Soft Classic', 'Acrylic', 'Medium (4)', '5.5mm', 'White', '4mm'),
+    (2, 'Loops and Threads Soft Classic', 'Acrylic', 'Medium (4)', '5.5mm', 'Black', '4mm');
+
+--
+-- PATTERN -- ONE TO MANY -> PROJECTS
+-- Name: pattern; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.pattern (
+    id SERIAL PRIMARY KEY,
+    project_id INT NOT NULL,
+    pattern_name VARCHAR(255), 
+    pattern_by VARCHAR(255),
+    pattern_url VARCHAR(255),
+    FOREIGN KEY (project_id) REFERENCES public.projects(id) ON DELETE CASCADE
+);
+
+--
+-- Name: pattern_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+--
+
+CREATE SEQUENCE public.pattern_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+    --
+-- Name: pattern_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: -
+--
+
+ALTER SEQUENCE public.pattern_id_seq OWNED BY public.pattern.id;
+
+--
+-- Name: pattern id; Type: DEFAULT; Schema: public; Owner: -
+--
+
+ALTER TABLE ONLY public.pattern ALTER COLUMN id SET DEFAULT nextval('public.pattern_id_seq'::regclass);
+
+--
+-- Insert Values into pattern table
+INSERT INTO public.pattern(project_id, pattern_name, pattern_by, pattern_url) 
+ VALUES 
+    (2, 'bunny with hat', 'bondinbuds', 'to be added');
+
+--
+-- OTHER MATERIALS -- MANY-TO-ONE -> PROJECTS
+-- Name: other_materials; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.other_materials (
+    id SERIAL PRIMARY KEY,
+    project_id INT NOT NULL,
+    hook_size VARCHAR(50),
+    safety_eyes VARCHAR(100),
+    stuffing VARCHAR(100),
+    FOREIGN KEY (project_id) REFERENCES public.projects(id) ON DELETE CASCADE
+);
+
+--
+-- Name: other_materials_id_seq; Type: SEQUENCE; Schema: public; Owner: -
+--
+
+CREATE SEQUENCE public.other_materials_id_seq
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+--
+-- Name: other_materials_id_seq; Type: SEQUENCE OWNED BY; Schema: public; Owner: -
+--
+
+ALTER SEQUENCE public.other_materials_id_seq OWNED BY public.other_materials.id;
+
+--
+-- Name: other_materials id; Type: DEFAULT; Schema: public; Owner: -
+--
+
+ALTER TABLE ONLY public.other_materials ALTER COLUMN id SET DEFAULT nextval('public.other_materials_id_seq'::regclass);
+
+--
+-- Insert Values into other_materials table
+INSERT INTO public.other_materials(project_id, hook_size, safety_eyes, stuffing) 
+ VALUES 
+    (2, '4mm', '10mm', '50 grams');
+
+--
+-- IMAGES -- MANY-TO-ONE -> PROJECTS
+-- Name: images; Type: TABLE; Schema: public; Owner: -
+--
+
+CREATE TABLE public.images (
```

Comment:
The images table is a many-to-one table with the projects table, but it is intended to be used for anything the user would like to include in their project photo album. However, to make the images table be more generic, would you suggest including the IDs of the other tables to be referenced in the images table?

Reply:
That makes more sense, thank you for the explanation! Since it is not for the mvp, I will make a note to look into adding the imageID array references when adding the "add image" feature. Thank you for the input.